doc/cookbook/source/examples/classifier/random_forest.rst

```diff
@@ -0,0 +1,38 @@
+============================
+Random Forest classification
+============================
+
+Random Forest is an ensemble learning method which implements multiple decision trees while training and performs prediction by using a combination rule on the outputs of individual decision trees.
+
+-------
+Example
+-------
+
+CDenseFeatures (here 64 bit floats aka RealFeatures) and :sgclass:`CMulticlassLabels` are created from training and test data file
+
+.. sgexample:: random_forest.sg:create_features
+
+A combination rule of :sgclass:`CMajorityVote` is created to be used as a combination rule. This and other combination rules are derived form the :sgclass:`CCombinationRule` .
+
+.. sgexample:: random_forest.sg:create_combination_rule
+
+Next an instance of :sgclass:`CRandomForest` is created. The parameters provided are the number of attributes to be chosen randomly to select from and the number of trees.
+
+.. sgexample:: random_forest.sg:create_instance
+
+Then we run the train random forest and apply it to test data, which here gives :sgclass:`CMulticlassLabels`.
+
+.. sgexample:: random_forest.sg:train_and_apply
+
+We can evaluate test performance via e.g. :sgclass:`CMulticlassAccuracy` as well as get the Out Of Bag Error.
+
+.. sgexample:: random_forest.sg:evaluate_accuracy
+
+----------
+References
+----------
+:wiki:`Random_forest`
+:wiki:`Out-of-bag_error`
+
+.. bibliography:: ../../references.bib
```

Comment:
can we have some references, e.g. on the first page here: http://civil.colorado.edu/~balajir/CVEN6833/lectures/cluster_lecture-2